docs/site/Decorators.md

```diff
@@ -132,56 +132,183 @@ accordingly or do a composition of them:
   ```
 
-  or
+  Writing the whole parameter specification is tedious, we've created shortcuts to define
+  the params with the pattern `@param.${in}.${type}(${name})`:
+
+  - in: the parameter location, it can be one of the following values: `query`, `header`, `path`.
+  - type: a [common name of OpenAPI primitive data type](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#data-types). 
+  - name: name of the parameter, it should be a `string`.
+
+  Take specifying a data from query as an example, a list of available shortcuts for can
+  be found in [API document](http://apidocs.loopback.io/@loopback%2fopenapi-v3/#param.query). 
 
+  An equivalent example using the shortcut decorator would be:
+  
   ```ts
-    // example 2: decorator `@param` applied on parameter level
     class MyController {
-      @get('/')
+      @get('/Pets/{category}')
       list(
-        @param(offsetSpec) offset?: number,
-        @param(pageSizeSpec) pageSize?: number,
+        @param.path.string('category') category: string,
+        @param.query.number('pageSizes') pageSize?: number,
       ) {}
     }
   ```
 
-  In the first example, we apply multiple decorators to a single declaration.
-  The order in which the decorators are called is important because a `@param`
-  decorator must be applied after an operation decorator.
-  To learn more about TypeScript decorator composition,
-  refer to [TypeScript Decorator Documentation](https://www.typescriptlang.org/docs/handbook/decorators.html)
+  You can find the specific usage in [Writing Controller methods](controllers.md#writing-controller-methods)
 
-  Please note method level `@param` and parameter level `@param` are mutually exclusive,
-  you can not mix and apply them to the same parameter.
+  *The parameter location cookie is not supported yet, see*
+  *https://github.com/strongloop/loopback-next/issues/997*
 
-  You can also use this pattern to make it quicker to define params: `@param.${in}.${type}(${name})`
+### RequestBody Decorator
 
-  - in: one of the following values: `query`, `header`, `path`, `formData`, `body`
-  - type: one of the following values:  `string`, `number`, `boolean`, `integer`
-  - name: a `string`, name of the parameter
+  Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/request-body-decorator.ts#L20-L79)
+
+  `@requestBody()` is applied to a controller method parameter to generate OpenAPI requestBody specification for it.
+  
+  *Only one parameter can be decorated by `@requestBody` per controller method.*
+
+  A typical OpenAPI requestBody spec contains properties `description`, `required`, and `content`:
+
+  ```ts
+    requestBodySpec: {
+    description: 'a user',
+    required: true,
+    content: {
+      'application/json': {...schemaSpec},
+      'application/text': {...schemaSpec},
+    },
+  }
+  ```
+
+  When the decorated argument is a custom type or a model, we recommend you define 
```

Comment:
`If`, not `When`. I'd also elaborate why we 'recommend' users to decorate the model

Reply:
I think you didn't understand the context here. 
As I have mentioned:

```
If the decorated argument is a custom type or a model, we recommend you define 
  a model class with `@model` and `@property`

//  some code snippet

Then a typical usage of `@requestBody` is specifying a parameter's type as the model class,
  and applying the decorator on it:
```

The contents are consistent.